torch/ao/quantization/qconfig_mapping.py

```diff
@@ -73,13 +74,24 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int):
         .set_object_type(torch.nn.BatchNorm2d, qconfig) \
         .set_object_type(torch.nn.BatchNorm3d, qconfig)
 
-def get_default_qconfig_mapping(backend="fbgemm", version=0):
+    # Use special observers for ops with fixed qparams
+    fixed_qparams_observer_to_qconfig: Dict[Any, QConfigAny] = {}
```

Comment:
what is this? I didn't see the code that modifies this dictionary

Reply:
Oh I forgot to populate it in the for loop below